src/PaymentIntent.php

```diff
@@ -14,7 +14,7 @@ class PaymentIntent extends PaymongoClient {
      * @param  mixed $metadata
      * @return mixed
      */
-    public function create($amount, $payment_method_allowed, $description = null, $metadata = null, $currency = 'PHP') {
+    public function create($amount, $payment_method_allowed, $payment_method_options, $description = null, $metadata = null, $currency = 'PHP') {
```

Comment:
if `payment_method_options` can be null as per code below, set the default value to `null`. Do the same for the `attachPaymentMethod()` method.

Reply:
Done